### PR TITLE
Hindi Translation

### PR DESCRIPTION
**Pull request by shrut1996**
I have added the Hindi Translation

### CODE REVIEW

**Comment by CoralineAda** [comment]
Hi there! Thanks so much for taking this on.

I see that you placed the translation in the 1/3/0 folder but you translated 1/4/0. Can you fix that?

Also the code_of_conduct.txt file needs the translation copied in as well.

I'm looking for a second native speaker to review the translation now.

Thanks again!

**Comment by shrut1996** [comment]
Hey there! It is my pleasure.
Sorry about that, I had been a bit careless. I have fixed the bugs now.

File: version/1/4/hi/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+आचार संहिता # योगदानकर्ता वाचा
+
+## हमारी प्रतिज्ञा
+
+एक खुला और स्वागत करते वातावरण को बढ़ावा देने के हित में, हम के रूप में
```
Comment (pksjce) [inline version/1/4/hi/code_of_conduct.md]: स्वागत करते वातावरण is really a google translation. The real phrase for a "welcoming environment" would be something like "सुस्वागतमय वातावरण "

Comment (pksjce) [inline version/1/4/hi/code_of_conduct.md]: s/ हम के रूप में / हम अर्थात

File: version/1/4/hi/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+योगदानकर्ताओं और देखरेख हमारी परियोजना में भागीदारी बनाने के लिए प्रतिज्ञा और
```
Comment (pksjce) [inline version/1/4/hi/code_of_conduct.md]: Maintainer => अनुरक्षक
So this becomes हम अर्थात  योगदानकर्ताओं और अनुरक्षक प्रतिज्ञा करते हैं की इस परियोजना एवं समुदाय में भाग लेना, एक उत्पीड़न मुक्त अनुभव रहेगा, बिना उम्र, शारीरिक आकार, विकलांगता, जातीयता, लिंग पहचान एवं अभिव्यक्ति, अनुभव का स्तर, राष्ट्रीयता, व्यक्तिगत उपस्थिति, जाति, धर्म, या यौन पहचान और उन्मुखीकरण के परवाह किये I

**Comment by shrut1996** [comment]
Thanks for the inputs @pksjce ! This looks more 'Hindi-ish' now :)

**Comment by CoralineAda** [comment]
Is this ready to merge?

**Comment by shrut1996** [comment]
@CoralineAda I reviewed the Hindi translation again and I believe it is ready to be merged.
Thanks.

**Comment by CoralineAda** [comment]
Can you please fix the merge conflict? Then I'm happy to push this out. Thanks!

**Comment by CoralineAda** [comment]
Thanks a lot! This is wonderful. ♥️

**Comment by shrut1996** [comment]
Good to go. Just some repetition of lines.
Thanks.

**Comment by pksjce** [comment]
Awesome!

**Pull request merged**